Bluepaper.md

```diff
@@ -44,9 +44,9 @@ In addition to these advancements in blockchain and token technology, Steem as a
 This is all done while providing users with three second confirmation times and zero fees on all transactions.
 All of this allows it to support the mission of bringing smart and social currency to publishers and community builders across the Internet.
 
-## Proof of Brain: Smart and Social Tokens
+## Proof of Brain: Mądre i społeczne tokeny
```

Comment:
Smart TV, Smartphony, to wszystko często tłumaczy się jako "inteligetny ...". Mądry mi tutaj nie do końca pasuje. Zastanawiam się też, czy przypadkiem tego "smart" nawet nie zostawić. Wszyscy teraz przecież mają "smartphony".

Z kolei social tłumaczył bym jako społecznościowy nie społeczny. Słowo społeczny kojarzy się raczej z zapomogami i programami typu 500+ :D

Co ty na to, by nazwać to:
- "Społecznościowe smart tokeny"?

Reply:
"Społecznościowe smart tokeny" - jak najbardziej masz rację, nie wiem czemu tam dałem "społeczne". "Smart" też jest ok, w zasadzie i tak się tej nazwy nie używa,ale nagłówek brzmi lepiej :)